helm.sh/_posts/2018-12-11-helm-hub.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Introducing the Helm Hub"
+permalink: "/blog/intro-helm-hub/"
+authorname: "Matt Farina"
+authorlink: "https://mattfarina.com"
+---
+
+Helm was designed with many distributed repositories in mind. Like Homebrew Taps and Debian APT repositories, Helm has the ability to add and work with many repositories. While the Helm [stable and incubator repositories](http://github.com/helm/charts) have been front and center from the beginning it was never our intent for these to be the only public repositories.
+
+With this in mind, we are delighted to announce the launch of the [Helm Hub](https://hub.helm.sh). This hub provides a means for you to find charts hosted in many distributed repositories hosted by numerous people and organizations. <!--more-->
+
+![Helm Hub](/assets/images/blog/helm-hub.png)
+
+Helm repositories can be hosted in many ways including as GitHub or Gitlab pages, in object storage, using [Chartmuseum](https://github.com/helm/chartmuseum), and via a service provider.
+
+If you have a chart repository you would like listed please head over to the [hub repository on GitHub](https://github.com/helm/hub) and follow the directions. The process is as simple as a pull request.
+
+The Helm Hub is powered by [Monocular](https://github.com/helm/monocular), one of the projects that has long been a part of Helm. This was originally crafted by Bitnami and Deis, now part of Microsoft. As the Helm Hub grows in complexity, Monocular will need to grow in its ability to handle many repositories and charts. This is an area that UI and UX designers looking to be part of the Helm and [CNCF](https://cncf.io) community can contribute and make a difference.
```

Comment:
Quick read, this is all great Matt! A few suggestions:

This is an area *where*...
or
...can contribute *to* and make a difference

Reply:
fixed